docker/src/main/java/brooklyn/entity/container/docker/DockerHostSshDriver.java

```diff
@@ -321,19 +313,51 @@ public void customize() {
                     .execute();
         }
 
+        //Create config for upstart
+        newScript(CUSTOMIZING)
```

Comment:
Give these scripts different names as `newScript(CUSTOMIZING + "-upstart")` and so on.